docs/plugins/kodiupdate.rst

```diff
@@ -16,6 +17,27 @@ which looks like this::
         user: kodi
         pwd: kodi
 
+With the basic configuration settings above, the ``kodiupdate`` plugin
+will notify Kodi to scan your entire music library after the import
+operation has completed. For large music libraries you may prefer to have
+Kodi only scan the directory where each album was imported. To enable
+this behavior, add `source` and `library` settings, like this:
```

Comment:
Can you add a brief description here of what exactly `source` is supposed to be? There's an example below, but a few words saying what it is in general (in terms Kodi users will understand) could be really useful.

Reply:
I've modified the documentation a bit to try to explain how the two extra options work. Seems wordy and unwieldy. :)